src/java.base/share/classes/java/lang/foreign/MemoryLayout.java

```diff
@@ -314,21 +384,23 @@ default long byteOffset(PathElement... elements) {
      * The returned method handle has the following characteristics:
      * <ul>
      *     <li>its return type is {@code long};</li>
-     *     <li>it has as zero or more parameters of type {@code long}, one for each <a href=#open-path-elements>open path element</a>
+     *     <li>it has at least one {@code long} leading parameter representing the base offset;</li>
```

Comment:
should be "it has one" e.g. not "at least one" right?

Reply:
I put at least one as a hint there might be more (dynamic) coordinates. But maybe it's not important. I'll simplify